packages/lib/src/index.js

```diff
@@ -2,4 +2,25 @@ import emotion from './emotion';
 
 const ccClassName = (...args) => args.join(' ');
 
-export default { ccClassName, emotion };
+/**
+ * Utility for making it easier to couple a React Component to a css selector.
+ * Useful when writing css selectors that rely on interactivity, i.e. :hover.
+ * Example:
+ *  const checkBoxWrapper = createDataProp('checkbox-wrapper');
+ *  // Used as selector:
+ *  css`&:hover ${checkboxWrapper.selector} { }`
+ *  // Used on React Component
+ *  <div {...checkboxWrapper.prop} />
+ * @param {string} name Name of element we want to reference.
+ */
+function createDataProp(name) {
+  const prefix = 'data-leafygreen-ui';
+  return {
+    prop: {
+      [prefix]: name,
+    },
+    selector: `[${prefix}="${name}"]`,
+  };
+}
+
+export default { emotion, ccClassName, createDataProp };
```

Comment:
Had a thought:

Aren't the following two expressions identical in this case?
```
export default { emotion, ccClassName, createDataProp };
```
vs
```
export { emotion, ccClassName, createDataProp };
```

I know this was here before, but should we really be using `export default` here?